core/src/org/sbml/jsbml/util/Maths.java

```diff
@@ -296,10 +296,21 @@ public static final double log(double number, double base) {
    * @return the rootExponent-th root of the radiant
    */
   public static final double root(double radiant, double rootExponent) {
-    if (rootExponent != 0) {
-      return Math.pow(radiant, 1 / rootExponent);
+    if (rootExponent == 0d) {
+      throw new ArithmeticException("Root exponent must not be zero.");
+    }
+    else if(rootExponent == 1d) {
+      return radiant;
+    }
+    else if(rootExponent == 2d) {
+      return Math.sqrt(radiant);
+    }
+    else if(rootExponent == 3d) {
+      return Math.cbrt(radiant);
+    }
+    else {
+      return Math.pow(radiant, 1d / rootExponent);
```

Comment:
The first `else` is not needed because the `if` case is only checked if no exception has been thrown. Also, the last `else` can be removed because it is only reached if no `return` statement has been executed.